app/helpers/hyrax/work_form_helper.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module Hyrax
+  # Helper for work form
+  module WorkFormHelper
+    def admin_set_options
+      return @admin_set_options.select_options if @admin_set_options
+
+      service = Hyrax::AdminSetService.new(controller)
+      Hyrax::AdminSetOptionsPresenter.new(service).select_options
+    end
+
+    # OVERRIDE FROM HYRAX. Adds in warning tab
+    def form_tabs_for(form:)
```

Comment:
@lamtu1 and I actually found you can shove this method into `app/helpers/hyrax_helper.rb` for less override.
https://github.com/osulp/Scholars-Archive/blob/main/app/helpers/hyrax_helper.rb#L26